cql3/expr/expression.cc

```diff
@@ -2189,9 +2189,6 @@ static cql3::raw_value evaluate(const function_call& fun_call, const evaluation_
 
     for (const expression& arg : fun_call.args) {
         cql3::raw_value arg_val = evaluate(arg, inputs);
-        if (arg_val.is_null()) {
-            throw exceptions::invalid_request_exception(format("Invalid null value for argument to {}", *scalar_fun));
-        }
```

Comment:
It's required indirectly, since we convert `x LIKE constant` to `func(x)`.

Functions handle NULL already, it's just this path that was broken (and isn't used in selection).